### PR TITLE
Upgrade to discord.js 10.0.1

### PR DESCRIPTION
**Pull request by jmariondev**
Based on the docs for discord.js, there are only a couple things needing to be changed to update:
<https://discord.js.org/#/docs/main/10.0.1/general/updating>

All that affected us was the different arguments for certain events.

This also fixes the license field of package.json to correctly be 'GPL-2.0'.

### CODE REVIEW

File: events/memberJoined.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ const moment = require('moment');
 const format = require('../momentFormat');
 
 module.exports = {
-  process: (bot, guild, member) => {
-    const userLogsChannel = guild.channels.find('name', 'user-logs');
-    const welcomeChannel = guild.channels.find('name', 'welcome-room');
+  process: (bot, member) => {
```
Comment (otter-computer) [inline events/memberJoined.js:5]: I assume this has been tested? Just want to make sure this was just some stupid code from me and there wasn't a specific (undocumented...) reason we were passing in the guild here.

File: package.json [hunk 2]
```diff
@@ -9,9 +9,9 @@
     "type": "git",
     "url": "git+https://github.com/gaymers-discord/DiscoBot.js"
   },
-  "license": "GNU",
+  "license": "GPL-2.0",
```
Comment (otter-computer) [inline package.json:12]: 👌

File: events/memberLeft.js [hunk 3]
```diff
@@ -3,9 +3,9 @@ const format = require('../momentFormat');
 
 module.exports = {
   // Logs a member leaving in #user-logs.
-  process: (bot, guild, member) => {
-    const userLogs = bot.channels.find('name', 'user-logs');
-    const welcomeRoom = bot.channels.find('name', 'welcome-room');
+  process: (bot, member) => {
```
Comment (otter-computer) [inline events/memberLeft.js:6]: See other comment about guild.

File: events/memberJoined.js (hunk 1 above)
Comment (jmariondev) [reply events/memberJoined.js:5]: That was part of their changes in 9 -> 10. You can derive `guild` from `member.guild` so they stopped passing it to the event handler. (See under the heading "Events" on the link in the OP.)

Comment (otter-computer) [reply events/memberJoined.js:5]: 👍

**Review by otter-computer — APPROVED** [review]

**Review by brandon-a — APPROVED** [review]

**Pull request merged**